packages/vuetify/src/components/VLayout/VLayoutItem.tsx

```diff
@@ -1,34 +1,46 @@
+// Styles
+import './VLayoutItem.sass'
+
 // Composables
 import { makeLayoutItemProps, useLayoutItem } from '@/composables/layout'
 
 // Utilities
-import { computed, defineComponent, toRef } from 'vue'
+import { defineComponent, toRef } from 'vue'
 import { makeProps } from '@/util'
 
 // Types
-import type { Prop } from 'vue'
+import type { PropType } from 'vue'
 
 export default defineComponent({
   name: 'VLayoutItem',
 
   props: makeProps({
     position: {
-      type: String,
+      type: String as PropType<'top' | 'right' | 'bottom' | 'left'>,
```

Comment:
I feel like this is common enough to have it's own type used throughout the framework.

Reply:
Maybe, but we probably also have a few variations of it. only top/bottom, only left/right, etc. It's easy enough to extract in the future if needed.